aii-opennebula/src/main/resources/tests/profiles/vm.pan

```diff
@@ -97,7 +105,8 @@ bind "/system/opennebula" = opennebula_vmtemplate;
 prefix "/system/opennebula";
 "vnet" = nlist(
     "eth0", "br100",
-    "eth1", "br101");
+    "eth1", "br101",
+    "eth2", "br101");
 
```

Comment:
@StephaneGerardVUB I have talked with @stdweird about this, we will implement both, you can use network interfaces or mac addrs, so the users can choose the best option that suit their needs. Now i'm fixing the schema that's a bit ugly... :sweat_smile: I hope to finish today